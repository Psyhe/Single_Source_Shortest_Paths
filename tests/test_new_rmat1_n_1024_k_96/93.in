1024 994 1003
994 32 168
996 425 181
997 18 208
997 33 10
