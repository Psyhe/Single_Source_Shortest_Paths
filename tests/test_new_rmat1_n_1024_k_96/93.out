171
1e+18
210
13
1e+18
1e+18
1e+18
1e+18
1e+18
1e+18
